docs/global_tools/concepts.md

```diff
@@ -91,11 +39,11 @@ exposed = { dotnet = 'dotnet\dotnet' }
 
 To increase efficiency, `pixi` uses *trampolines*—small, specialized binary files that manage configuration and environment setup before executing the main binary. The trampoline approach allows for skipping the execution of activation scripts that have a significant performance impact.
 
-When you execute a global install binary, a trampoline performs the following sequence of steps:
+When you execute a globally installed executable, a trampoline performs the following sequence of steps:
 
-* Each trampoline first reads a configuration file named after the binary being executed. This configuration file, in JSON format (e.g., `python.json`), contains key information about how the environment should be set up. The configuration file is stored in `.pixi/bin/trampoline_configuration`.
+* Each trampoline first reads a configuration file named after the binary being executed. This configuration file, in JSON format (e.g., `python.json`), contains key information about how the environment should be set up. The configuration file is stored in `[$PIXI_HOME](../reference/environment_variables.md)/bin/trampoline_configuration`.
```

Comment:
Correct Markdown syntax would be:

> ```md
> [`$PIXI_HOME`](../reference/environment_variables.md)`/bin/trampoline_configuration`
> ```

which renders to

> [`$PIXI_HOME`](../reference/environment_variables.md)`/bin/trampoline_configuration`

Same applies to the occurences further below.

Reply:
Alternatively you could revert to plain HTML which allows for what you actually want:

> ```html
> <code><a href='../reference/environment_variables.md'>$PIXI_HOME</a>/bin/trampoline_configuration</code>
> ```

which renders to:

> <code><a href='../reference/environment_variables.md'>$PIXI_HOME</a>/bin/trampoline_configuration</code>

Reply:
Interestingly, mkdocs, doesn't seem to love relative links with html